### PR TITLE
added README.md

### PR DESCRIPTION
**Pull request by Rudiachenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
...
++ Can create / delete orders; 
++ Can create / delete shopping cart;
++ Can delete users;
++ Have all functional of USER
+##### ***USER:***
++ Can add a product to the shopping cart;
++ Can place an order;
++ Can view their orders.
+
+##### ***This project used technologies such as:***
```
Comment (andromash) [inline README.md]: Move this part before Administrator

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
++ Java 8
++ JDBC
++ Tomcat
++ Servlets
++ MqSQL
```
Comment (andromash) [inline README.md]:
```suggestion
+ MySQL
```

File: README.md [hunk 3]
```diff
@@ -1 +1,28 @@
...
++ OOP, SOLID, RBAC principles, n-tier architecture.
```
Comment (andromash) [inline README.md]: I think you need to describe this at the start and try to tell more details

**Review by andromash — COMMENTED** [review]
Add configuration info after technologies. How does the user launch this project? what SQL script he needs to run? etc.

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**